usuarios/post-usuarios.spec.js

```diff
@@ -1,20 +1,43 @@
-const faker = require('faker')
+const user = require('../utils/userData')
 const testServer = require('../utils/testServer')
+const rota = require('../utils/rotas')
 
-const rotaUsuarios = '/usuarios'
+var usuario
```

Comment:
```suggestion
let usuario
```

faz mais sentido usar o let
ref https://imasters.com.br/front-end/let-const-e-var-nao-e-tudo-mesma-coisa